src/content/reducers/types.js

```diff
@@ -1,14 +1,23 @@
+// @flow
+
 import type { Summary } from '../../common/summarize-profile';
-import type { Action, ExpandedSet, CallTreeFiltersPerThread, DataSource } from '../actions/types';
+import type { Action, ExpandedSet, CallTreeFiltersPerThread, DataSource, ProfileSelection } from '../actions/types';
 import type { Milliseconds, StartEndRange } from '../../common/types/units';
-import type { Profile, ThreadIndex } from '../../common/types/profile';
+import type { IndexIntoMarkersTable, IndexIntoFuncTable, Profile, ThreadIndex } from '../../common/types/profile';
 
 export type Reducer<T> = (T, Action) => T;
 
+export type RequestedLib = { pdbName: string, breakpadId: string };
+export type SymbolicationStatus = 'DONE' | 'SYMBOLICATING';
+export type ThreadViewOptions = {
+  selectedFuncStack: IndexIntoFuncTable[],
+  expandedFuncStacks: Array<IndexIntoFuncTable[]>,
+  selectedMarker: IndexIntoMarkersTable | -1,
+};
 export type ProfileViewState = {
   viewOptions: {
-    threads: ThreadViewOptions[],
```

Comment:
I believe this was a mistake because this property is present in the unprocessed profile.